README.md

```diff
@@ -47,6 +47,20 @@ import { ref, reactive, defineComponent } from 'vue-demi'
 
 Publish your plugin and all is done!
 
+### Extra APIs
+
+Vue Demi provides extra APIs `isVue2` `isVue3` to help distinguishing user versions.
+
+```ts
+import { isVue2 } from 'vue-demi'
+
+if (isVue2) {
+  // vue 2 only logics
+} else {
+  // vue 3 only logics
+}
+
```

Comment:
Can you help add the ending backquote? Not able to do that by review suggestions 😅